src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -649,6 +679,95 @@ def get_observation(self, event: Event) -> pd.DataFrame:
         for col in ["employer_address_id", "employer_name"]:
             df_w2[col] = df_w2["employer_id"].map(emp[col])
 
+        # re-initialize income-to-date series for next year of income counting
+        self.income_last_year = self.income_this_year
```

Comment:
Also, I just remembered that I think the whole point (or one of the main points) of combining these observers was so that the base TaxObserver would manage income, not the W2.

Reply:
I was thinking of the income as fundamentally a part of the W2, but we can refactor this in a future PR if you think it will be clearer to move it into the TaxObserver.